tests/python/YapfBearTest.py

```diff
@@ -22,6 +24,15 @@ def test_valid(self):
                             ["x = {  'a':37,'b':42,\n", "'c':927}\n", '\n',
                              "y = 'hello ''world'\n"], valid=False)
 
+    def test_valid_python_2(self):
+        self.check_validity(self.uut, ['print 1\n'], valid=True)
```

Comment:
How come this works ?
Doesnt the documentation say "if you format Python 3 code with YAPF, run YAPF itself under Python 3 (and similarly for Python 2)." ?